### PR TITLE
Some changes to antag and ERT text

### PR DESCRIPTION
**Pull request by rhicora**
Copy changes to antags and ERTs, as well as some radio channels.

### CODE REVIEW

File: code/controllers/communications.dm [hunk 1]
```diff
@@ -126,20 +126,20 @@ var/const/MED_I_FREQ = 1485
...
 	"Science"		= SCI_FREQ,
 	"Command"		= COMM_FREQ,
 	"Medical"		= MED_FREQ,
 	"Engineering"	= ENG_FREQ,
 	"Security" 		= SEC_FREQ,
-	"Response Team" = ERT_FREQ,
-	"Special Ops" 	= DTH_FREQ,
-	"Mercenary" 	= SYND_FREQ,
+	"Epsilon-11" 	= ERT_FREQ,
+	"Nu-7" 			= DTH_FREQ,
```
Comment (JakeDaBoss) [inline code/controllers/communications.dm:136]: We use Hammer-Down for deathsquad.

**Review by JakeDaBoss — CHANGES_REQUESTED** [review]

File: code/controllers/communications.dm (hunk 1 above)
Comment (rhicora) [reply code/controllers/communications.dm:136]: Nu7 is Hammer Down. : )

Comment (rhicora) [reply code/controllers/communications.dm:136]: This pull makes that explicit actually, it was just "death commando" before.

image

**Comment by rhicora** [comment]
This branch is fine, no changes needed afaik.

**Review by JakeDaBoss — APPROVED** [review]

**Pull request merged**